requirements.txt

```diff
@@ -12,7 +12,8 @@ navmazing==1.2.2
 productmd==1.38
 pyotp==2.9.0
 python-box==7.1.1
-pytest==8.0.2
+pytest==8.1.1
+pytest-order==1.2.0
```

Comment:
@omkarkhatavkar is adding pytest-order dependancy from master is intentional here?
PR adding here was never cherrypicked to this older versions https://github.com/SatelliteQE/robottelo/pull/13068

Reply:
https://github.com/SatelliteQE/robottelo/pull/13068 has much more than the pytest-order==1.2.0, It's looks like a healthy plugin can be used. If this is something we dont want I can remove it.

Reply:
yes, it is indeed healthy plugin, but it will be unused in 6.14.z and below branches anyway as there's no tests using it, that's why we can either remove it or keep it, its upto you @omkarkhatavkar @jyejare

Reply:
I think we should pull all the requirements until 6.12.z regardless to avoid the conflicts in that file for others / PRs.

Reply:
right, thanks for confirming, we can merge this